articles/tutorials/qft-tutorial.md

```diff
@@ -1,62 +1,82 @@
+Note that this low-level view of quantum information processing is often
+described in terms of "[quantum
+circuits](xref:microsoft.quantum.concepts.circuits)," which represent the
+sequential application of gates to specific qubits of a system.
 
-Thus, the single- and multi-qubit operations we sequentially apply can be readily represented in a "circuit diagram."
-In our case, we will define a Q# operation to perform the full three-qubit quantum Fourier transform, which has the following representation as a circuit:
+Thus, the single- and multi-qubit operations we sequentially apply can be
+readily represented in a "circuit diagram." In our case, we will define a Q#
+operation to perform the full three-qubit quantum Fourier transform, which has
+the following representation as a circuit:
 
-<br/>
-<img src="./qft_full.PNG" alt="Three qubit quantum Fourier transform circuit diagram" width="600">
+<br/> <img src="./qft_full.PNG" alt="Three qubit quantum Fourier transform
+circuit diagram" width="600">
 
 ## Prerequisites
 
-* [Install](xref:microsoft.quantum.install) the Quantum Development Kit using your preferred language and development environment
-* If you already have the QDK installed, make sure you have [updated](xref:microsoft.quantum.update) to the latest version
-
+* [Install](xref:microsoft.quantum.install) the Quantum Development Kit using
+  your preferred language and development environment
+* If you already have the QDK installed, make sure you have
+  [updated](xref:microsoft.quantum.update) to the latest version
 
 ## In this tutorial, you'll learn how to:
 
 > [!div class="checklist"]
+>
 > * Define quantum operations in Q#
-> * Set up the classical host program to call and simulate Q# operations
+>* Call and simulate Q# operations with the command line
+>  application for Q#.
 > * Simulate a quantum operation from qubit allocation to measurement output
-> * Observe how the quantum system's simulated wavefunction evolves throughout the operation
+> * Observe how the quantum system's simulated wavefunction evolves throughout
+>   the operation
+
+Applications developed with Microsoft's Quantum Development Kit typically
+consists of:
 
-Applications developed with Microsoft's Quantum Development Kit typically consists of two parts:
-1. A quantum program, implemented using the Q# quantum programming language, which is then invoked by the classical host program to run on a quantum computer or quantum simulator. These consist of 
-    - Q# operations: subroutines containing quantum operations, and 
-    - Q# functions: classical subroutines used within the quantum algorithm.
-2. A classical program, implemented in a classical programming language like Python or C#, that serves as the main entry point and will invoke Q# operations when it wants to execute a quantum algorithm.
+- Q# operations: subroutines containing quantum operations, and 
```

Comment:
Yeah, this was part of a bigger explanation that explained host programs. Now it looks a bit out of place, and since we don't use functions in this tutorial I think it is better to remove that.